Source/DafnyRuntime/DafnyRuntime.cs

```diff
@@ -652,18 +653,21 @@ public class MultiSet<T> : IMultiSet<T> {
     public bool IsEmpty { get { return occurrencesOfNull == 0 && dict.IsEmpty; } }
 
     public int Count {
-      get { return (int)ElementCount(); }
+      get { return (int)ElementCount; }
     }
     public long LongCount {
-      get { return (long)ElementCount(); }
+      get { return (long)ElementCount; }
     }
-    private BigInteger ElementCount() {
-      // This is inefficient
-      var c = occurrencesOfNull;
-      foreach (var item in dict) {
-        c += item.Value;
+
+    public BigInteger ElementCount {
+      get {
+        // This is inefficient
```

Comment:
If computing `ElementCount` is `inefficient`, shouldn't it be a method instead of a property?

Reply:
Maybe, but the other two properties just cast down the exact solution computed here, so there's a precedent.